tests/test_input_damages.py

```diff
@@ -0,0 +1,1287 @@
+                        "rcp",
+                        "model",
+                        "year",
+                        "region",
+                        "scaling",
+                        "valuation",
+                    ],
+                    np.full((2, 1, 2, 2, 2, 2, 2, 4, 2), 0),
+                ),
+                "monetized_deaths": (
+                    [
+                        "gcm",
+                        "batch",
+                        "ssp",
+                        "rcp",
+                        "model",
+                        "year",
+                        "region",
+                        "scaling",
+                        "valuation",
+                    ],
+                    np.full((2, 1, 2, 2, 2, 2, 2, 4, 2), 1),
+                ),
+                "monetized_histclim_deaths": (
+                    [
+                        "gcm",
+                        "batch",
+                        "ssp",
+                        "rcp",
+                        "model",
+                        "year",
+                        "region",
+                        "scaling",
+                        "valuation",
+                    ],
+                    np.full((2, 1, 2, 2, 2, 2, 2, 4, 2), 2),
+                ),
+            },
+            coords={
+                "batch": (["batch"], [b]),
+                "gcm": (["gcm"], ["ACCESS1-0", "GFDL-CM3"]),
+                "model": (["model"], ["IIASA GDP", "OECD Env-Growth"]),
+                "rcp": (["rcp"], ["rcp45", "rcp85"]),
+                "region": (["region"], ["USA.test_region", "ZWE.test_region"]),
+                "scaling": (
+                    ["scaling"],
+                    ["epa_scaled", "epa_iso_scaled", "epa_popavg", "epa_row"],
+                ),
+                "ssp": (["ssp"], ["SSP2", "SSP3"]),
+                "valuation": (["valuation"], ["vsl", "vly"]),
+                "year": (["year"], [2010, 2099]),
+            },
+        )
+
+        d = os.path.join(tmp_path, "mortality_in")
+        if not os.path.exists(d):
+            os.makedirs(d)
+        infile = os.path.join(d, f"mortality_damages_batch{b}.zarr")
+
+        ds_in.to_zarr(infile)
+
+    prep_mortality_damages(
+        gcms=["ACCESS1-0", "GFDL-CM3"],
+        paths=[
+            os.path.join(tmp_path, f"mortality_in/mortality_damages_batch{b}.zarr")
+            for b in [6, 9]
+        ],
+        vars={
+            "delta_costs": "monetized_costs",
+            "delta_deaths": "monetized_deaths",
+            "histclim_deaths": "monetized_histclim_deaths",
+        },
+        outpath=os.path.join(tmp_path, "mortality_out"),
+        mortality_version=version_test,
+        path_econ=os.path.join(tmp_path, "econvars_for_test", "econvars_for_test.zarr"),
+    )
+
+    ds_out_actual = xr.open_zarr(
+        os.path.join(
+            tmp_path,
+            "mortality_out",
+            f"impacts-darwin-montecarlo-damages-v{version_test}.zarr",
```

Comment:
Is this filename hard-coded elsewhere? Not sure where it came from

Reply:
This is the file name we assigned to the output of `prep_mortality_damages` function
https://github.com/ClimateImpactLab/dscim/blob/802f129d4bc2c13cc1520b24df7b843721285015/src/dscim/preprocessing/input_damages.py#L749-L753